Library/Homebrew/installed_dependents.rb

```diff
@@ -26,7 +26,7 @@ def find_some_installed_dependents(kegs, casks: [])
     kegs_by_source = kegs.group_by do |keg|
       # First, attempt to resolve the keg to a formula
       # to get up-to-date name and tap information.
-      f = keg.to_formula
+      f = keg.to_formula(prefer_stub: true)
```

Comment:
Same deal with kegs: they should be able to use the formula file they have internally.